src/dioptra/restapi/v1/models/service.py

```diff
@@ -550,14 +571,23 @@ def create(
 
         ml_model.resource.children.append(new_version.resource)
         db.session.add(new_version)
+        db.session.flush()
 
-        if commit:
-            db.session.commit()
-            log.debug(
-                "Model registration successful",
-                model_id=ml_model.resource_id,
-                name=ml_model.name,
+        mlflow_model_name = f"resource_{ml_model.resource_id:09d}"
+        try:
+            self._mlflow_client.create_model_version(
+                mlflow_model_name, source=artifact.uri
```

Comment:
How are we linking our version numbers with the ones in the MLflow database? Are we assuming a 1-to-1 correspondence in version numbers? What do we do if something happens and the version numbers suddenly are off-by-1 (or more) between the databases?